src/core_plugins/input_control_vis/public/components/editor/field_select.js

```diff
@@ -48,7 +48,13 @@ export class FieldSelect extends Component {
       return;
     }
 
-    const indexPattern = await this.props.getIndexPattern(indexPatternId);
+    let indexPattern;
+    try {
+      indexPattern = await this.props.getIndexPattern(indexPatternId);
+    } catch (err) {
+      // index pattern no longer exists
+      return;
```

Comment:
Is that the only source of exceptions here? Seems like a more concrete check might be appropriate, but that might be overkill, too. Just a thought.

Also, might be worth moving the try/catch fetch index pattern out into a function so that you can use a const for `indexPattern` and then do an if null -> return or something. Not important, but I prefer avoiding lets where possible, since they add mental context when reading code. With consts, I can see their declaration and then largely forget about them.